cmd/nerdctl/internal_oci_hook.go

```diff
@@ -41,5 +41,6 @@ func internalOCIHookAction(clicontext *cli.Context) error {
 		dataStore,
 		clicontext.String("cni-path"),
 		clicontext.String("cni-netconfpath"),
+		clicontext.String("pidfile"),
```

Comment:
This flag doesn't seem to be consumed.

Better to use labels rather than flags
https://github.com/containerd/nerdctl/blob/d5b9f42a05f377143da35baf29615b55917182e1/cmd/nerdctl/run.go#L803

Reply:
thank you, fixed